docs/extension-releasing.md

```diff
@@ -1,10 +1,49 @@
 # Extension Releasing
 
-Gemini CLI extensions can be distributed as pre-built binaries through GitHub Releases. This provides a faster and more reliable installation experience for users, as it avoids the need to clone the repository and build the extension from source.
+There are two primary ways of releasing extensions to users:
 
-## Asset naming convention
+- [Git repository](#releasing-through-a-git-repository)
+- [Github Releases](#releasing-through-github-releases)
 
-To ensure Gemini CLI can automatically find the correct release asset for each platform, you should follow this naming convention. The CLI will search for assets in the following order:
+Git repository releases tend to be the simplest and most flexible approach, while GitHub releases can be more efficient on initial install as they are shipped as single archives instead of requiring a git clone which downloads each file individually. Github releases may also contain platform specific archives if you need to ship platform specific binary files.
+
+## Releasing through a git repository
```

Comment:
The #1 most important piece of info here is that you can upload your extension from a github repo and it will immediately just work if the repo is public-- can you highlight that and then add a sub-section for the info about refs?